ui/admin-portal/src/app/model/candidate-opportunity.ts

```diff
@@ -73,6 +73,7 @@ export enum CandidateOpportunityStage {
   jobOfferRetracted = "Closed. Job offer retracted",
   relocatedNoJobOfferPathway = "Closed. No job offer stream (Canadian pathway stream - based on skills not job offer)",
   trainingNotCompleted = "Closed. Training not completed",
+  candidateMistakenlyAddedToList = "Closed. Candidate mistakenly added to list"
```

Comment:
Can we rename this to something more related to jobs. 
I suggest

candidateMistakenProspect = "Closed. Candidate was mistakenly proposed as a prospect for the job."

Also can we modify the ordering.

Same with candidate portal version

Reply:
Amended per your suggestions, @camerojo!